packages/app/src/app/overmind/namespaces/userNotifications/actions.ts

```diff
@@ -189,6 +189,11 @@ export const openTeamAcceptModal = (
   state.currentModal = 'teamInvite';
 };
 
+export const openNotificationPreferencesModal = ({ state }: Context) => {
```

Comment:
Is this the right place to put this? It seemed sensible since this is where the other notification-related actions lived.